src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -244,4 +245,40 @@ describe("provider", () => {
       );
     });
   });
+
+  describe("web3 compatibility", () => {
+    const networkId = 1234;
+    let provider: EthereumProvider;
+    let web3: Web3;
+    let accounts: string[];
+
+    beforeEach(async () => {
+      provider = await getProvider({
+        chain: { networkId },
```

Comment:
Do we need to set a `networkId` here? If not I think it should be removed.

Reply:
Removed.